### DISCUSSION TITLE
Silent install for Microsoft Intune

### DISCUSSION THREAD

**Discussion by coumeto** [Devolutions/UniGetUI#765] [General]
Hi ! 

I'm trying to package WingetUI to win32 app for Intune.

I use the argument /verysilent for silent install, but at the end of the installation the application automatically opens.

Do you know if there is a way to avoid that ? 

I've tried to do it with winget install but same problem, the app launch automatically at the end of installation.

Thanks a lot for your help !

**Comment by marticliment** [comment]
I don't know if there are any command line parameters to not start files on inno setup. if not, tell me and I will disable autorun when very silent

**Reply by coumeto** [reply]
Hi ! Thanks for you answer. It looks like it's impossible to do with command line parameters : https://groups.google.com/g/innosetup/c/ZB6dut8rmvg?pli=1 

So if you can modify the exe with the /verysilent parameters it would be very cool.

Thanks a lot :)

**Reply by marticliment** [reply]
Please see #769

**Reply by marticliment** [reply]
Hello, 

From version 1.6.4. and up (coming in the following hour), you will be able to disable autorun by using the parameter "/NoAutoStart" on the installer

Edit: Version 1.6.4 has been released

**Comment by coumeto** [comment]
Oh nice thank you very much !

Le jeu. 30 mars 2023 à 18:02, Martí Climent ***@***.***> a
écrit :
